Code/Source/solver/fs.cpp

```diff
@@ -303,9 +303,9 @@ void init_fs_msh(const ComMod& com_mod, mshType& lM)
     lM.fs[0].Nx  = lM.Nx;
   }
 
-  bool flag = (lM.eType == ElementType::HEX20) || (lM.eType == ElementType::HEX27) || (lM.eType == ElementType::WDG);
+  bool flag = (lM.eType == ElementType::HEX20) || (lM.eType == ElementType::HEX27);
```

Comment:
No need for a `flag` variable, make this an if- statement.

Reply:
I removed this flag and the following warning, since it was unnecessary